src/components/sidebar/lineView/LineHeaderTable.tsx

```diff
@@ -60,42 +72,41 @@ class LineTopicTable extends React.Component<ILineTopicListProps> {
                                 </th>
                                 <th className={s.columnHeader} />
                             </tr>
-                            {this.props.lineTopics.map(
-                                (lineTopic: ILineTopic, index: number) => {
-                                    const isCurrentLineTopic = _.isEqual(
-                                        this.props.currentLineTopic,
-                                        lineTopic
+                            {this.props.lineHeaders.map(
+                                (lineHeader: ILineHeader, index: number) => {
+                                    const isCurrentLineHeader = _.isEqual(
+                                        this.props.currentLineHeader,
+                                        lineHeader
                                     );
                                     return (
                                         <tr
                                             key={index}
                                             className={
-                                                isCurrentLineTopic
-                                                    ? s.lineTopicRowHighlight
+                                                isCurrentLineHeader
```

Comment:
Use classnames? 
```
className={classnames(
  s.lineHeaderRow,
  isCurrentLineHeader ? s.lineHeaderRowHighlight : null
)}
```

Reply:
There is no such class as .lineHeaderRow?